packages/medusa/src/loaders/module.ts

```diff
@@ -46,15 +60,22 @@ export default async ({
     } catch (err) {
       if (resolution.definition.isRequired) {
         throw new Error(
-          `Could not resolve required module: ${resolution.definition.label}`
+          `Could not resolve required module: ${resolution.definition.label}.`
         )
```

Comment:
thought: Instead of throwing a generic error, we could throw the error from the module loader? I've added that in the event bus module PR, so would love to hear what you think. Would make it easier to debug :)

Reply:
So do:
```ts
   if (resolution.definition.isRequired) {
        logger.warn(`Could not resolve required module: ${resolution.definition.label}`)
        throw err // throw error from module loader on required modules
      }
```